apps/web/src/services/dashboards/components/dashboard-detail/DashboardVariablesV2.vue

```diff
@@ -59,12 +62,16 @@ const state = reactive({
         _defaultDashboardVariables.region = MANAGED_DASHBOARD_VARIABLES_SCHEMA.properties[MANAGED_DASHBOARD_VARIABLE_MODEL_INFO_MAP.region.key];
         return _defaultDashboardVariables;
     }),
+    newGlobalVariables: computed<DashboardGlobalVariable[]>(() => {
+        const properties = dashboardDetailGetters.dashboardVarsSchemaProperties;
+        return orderBy(Object.values(properties), ['name'], ['asc']);
+    }),
```

Comment:
nitpick: Preset vars need to be sorted first! Here is the code I wrote.

```typescript
const _presetKeys: string[] = Object.keys(DOMAIN_DASHBOARD_VARS_SCHEMA_PRESET.properties);
const _presetItems = _tableItems.filter((d) => _presetKeys.includes(d.key));
const _customItems = _tableItems.filter((d) => !_presetKeys.includes(d.key));
return [
    ...orderBy(_presetItems, 'name', 'asc'),
    ...orderBy(_customItems, 'name', 'asc'),
];

```

Reply:
Also, how about removing the `new` prefix and deleting `variableProperties` state?